codecov.yml

```diff
@@ -0,0 +1,27 @@
+codecov:
+  require_ci_to_pass: no
+  notify:
+    wait_for_ci: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "50...70"
+  status:
+    default_rules:
+      flag_coverage_not_uploaded_behavior: exclude
+    project:
+      target: auto
```

Comment:
missing
default:

Reply:
Thanks @gaborkb - @mjmvisser the codecov.yml section in the doc was updated on Monday, so these changes did not make it in, apologies

Regarding the 'pull request base' error, in order to get the PR bot and status check(s) to work you need to run the coverage job (and codecov upload) in both base and head changesets, so that it can compare between the two.

You would need to branch off a branch that has the codecov job added and PR into it